app/policies/project_policy.rb

```diff
@@ -33,14 +33,20 @@ def unarchive?
   alias ownership? share?
 
   class Scope < Scope
+    def project_list
```

Comment:
What do you think about renaming it to `permitted_project_ids`? This method returns the ids of the projects the user is allowed to see, right?